tools/install.sh

```diff
@@ -41,6 +41,20 @@ then
 	shift
 fi
 
+# ------------------- Second, check OS  ----------------------
+OS=""
+if lsb_release -sid | grep "CentOS"
```

Comment:
I think it's better to be dependent on having a apt or yum command, then on specific os name.

Reply:
I agree, now scripts checks whether yum or apt-get exists.